src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -8360,6 +8360,363 @@ .maxstack  2
   IL_003a:  blt.s      IL_000e
   IL_003c:  ret
 }
+");
+        }
+
+        [Fact]
+        public void ExpressionVariable_InConstructorInitializer()
```

Comment:
>ExpressionVariable_InConstructorInitializer [](start = 20, length = 43)

There is already test `public void OutVar()` that's similar. Could you rename it to `ExpressionVariable_InStatement` or something like that and move these tests close together?